File: common/src/main/java/org/chilly/common/dto/PlaceDto.java
package org.chilly.common.dto;

import lombok.*;

import java.io.Serializable;
import java.util.List;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PlaceDto implements Serializable {
    Long id;
    String name;
    String address;
    String website; /*nullable*/
    String yPage; /*nullable*/
    Double rating; /*nullable*/
    List<String> images;
    String phone; /*nullable*/
    List<String> social; /*nullable*/
    List<String> openHours; /*nullable*/
    Double latitude;
    Double longitude;
    Long ownerId; /*nullable*/
}

File: places-svc/src/main/kotlin/com/chilly/places_svc/dto/PlaceDto.kt
package com.chilly.places_svc.dto

data class PlaceDto(
    val id: Long,
    val name: String,
    var address: String? = null,
    var website: String? = null,
    var yPage: String? = null,
    var rating: Double? = null,
    var images: List<String>? = null,
    var phone: String? = null,
    var social: List<String>? = null,
    var openHours: List<String>? = null
)
